src/main/scala/cognite/spark/v1/RawTableRelation.scala

```diff
@@ -76,6 +77,32 @@ class RawTableRelation(
     cursors.map(rawClient.filterOnePartition(filter, _, limit))
   }
 
+  def pullFromKeysSeq(
+      keys: List[String],
+      get: String => IO[RawRow]
+  ): Pull[IO, RawRow, Unit] =
+    keys match {
```

Comment:
Very cool! But, I think we can do something like `Stream.emit(keys).parEvalMap(getRowByKey)`. See comment below.